datafusion/sqllogictest/test_files/arrow_typeof.slt

```diff
@@ -237,6 +237,31 @@ drop table foo
 
 ## Timestamps: Create a table
 
+statement ok
```

Comment:
The timestamp format has changed (improved!) so let's also add tests for the new format